src/App.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, createRef } from "react";
```

Comment:
Why is this file (and most others) renamed from `.jsx` to `.js`?

Reply:
We should keep these files "jsx" as code editors like Visual Studio Code recognize these files specifically as "ReactJS" files rather than JavaScript files, and may have special rules for them.

Reply:
`create-react-app` started to migrate all files to just use `.js` extensions. Read more:

https://github.com/facebook/create-react-app/issues/87#issuecomment-234627904

If you create a new react app with `yarn create react-app` (which our frontend does) now, no more `.jsx`.

Also cc to @shawntabrizi